### ISSUE TITLE
Go Paths & AutoUpdate

### ISSUE THREAD

**Issue by joesixpack** [ElrondNetwork/elrond-go-scripts-v2#9]
Don't hardcode the path for your manual install of go binary for those of use that use PPA's.  Ubuntu's system default path for go is not /usr/local/go, its /usr/lib/go.  What you want to respect is any existing $GOROOT env variable for finding the go binary or set it during manual install and use that elsewhere.

![image](https://user-images.githubusercontent.com/23145642/70017336-b0b11000-1537-11ea-927f-2f40e386ed8b.png)

**Comment by koen84** [comment]
Would that go PPA not have to update $PATH itself to what it needs ?

**Comment by koen84** [comment]
I also think the middle export line is obsolete, but that's not critical.

**Comment by joesixpack** [comment]
The problem is go wants $GOROOT set correctly.  When the start script manual installed go directly to /usr/local/bin, it conflicted with the $GOROOT that was already in my .profile (that I copied over from another machine, but before I installed go via PPA), so it wouldn't run.

So the install script needs to set the $GOROOT env at least, even though my preference is it should install go from the PPA  (and what I use so go is kept updated), but that might be a security risk.  AFAICR, the go install locations from the PPA match go from the official Ubuntu respository.
